ansible/roles/test/tasks/continuous_link_flap.yml

```diff
@@ -54,13 +54,14 @@
       delegate_to: localhost
       tags: always
 
-    - debug: msg="{{ device_conn }}"
+    - name: Get up interface facts
+      set_fact: up_ports="{{minigraph_ports | list}}"
```

Comment:
Changes look good to me.
Kindly paste the values for {{ device_conn }} and {{minigraph_ports}} here once, so that it is documented that there can be ports on the switch, which can not be part of the topology.